README.md

```diff
@@ -1 +1,161 @@
+  <ul>
+    <li><a href="https://ruby-doc.org/3.2.2/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Record all expenses by creating a category
+- List all transactions associated with a category
+- All users to visualize how much money they spent on a category
+
+## Live Demo
+
+- [Live Demo](https://budget-tracker-pzur.onrender.com/)
+- [Project video Link](https://www.loom.com/share/752c6b57b76b42ea8b5528a24d00c47b?sid=39967d9d-27d2-4d08-ab28-12008daed00f)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor (Vs Code) or any editor of your choice
+
+- Terminal
+
+- Install Ruby on your computer
+
+- Install Rails on your computer
+
+### Setup
+
+Clone this repository by using the command line to your desired folder:  
+
+```sh
+git clone  https://github.com/Microverse-Fullstack-Program/rails-budget-app
+
+cd rails-budget-app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+cd rails-budget-app
+ bundle install (install all dependencies)
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+rails server
+```
+
+### Test
+
+```sh
+  rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+
+👤 **Chere Lemma**
+
+- GitHub: [Chere Lemma](https://github.com/cherelemma)
+- LinkedIn: [Chere (Urgaya) Lemma](https://www.linkedin.com/in/chere-lemma27211613/)
+- Twitter: [@Chere21271613](https://twitter.com/Chere21271613)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [] Implement some UX improvements: like transitions and/or animations, etc.
+- [] Create a decent desktop design for the webapp.
+- [] Implement the left side menu to improve the navigability of the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Microverse-Fullstack-Program/rails-budget-app/issues)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Leave a star if you like the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+design credits go to:
+[@Gregoire Vella](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) - author of design
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- The link is broken, kindly fix this to make your readme more professional.

Reply:
- [ ] This issue has not yet been resolved. you only renamed the file, the link is still broken. please, fix this:

![image](https://github.com/Microverse-Fullstack-Program/rails-budget-app/assets/59397159/ff0ef1e5-43a1-4cd9-ac04-0e023434af95)

Reply:
@cherelemma 
You have not properly resolved this. The "Your Name" placeholder should have your full name and the "APP TYPE" place holders should have the name of your app. Fix this, I am waiting.
<img width="1097" alt="Screenshot 2023-07-29 at 23 45 48" src="https://github.com/Microverse-Fullstack-Program/rails-budget-app/assets/58692590/ef59124a-5eaf-45e2-ba3d-0f3cd0a2b687">